File: room_database/src/main/java/com/example/room_database/MainActivity.java
package com.example.room_database;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import movie_database.MovieScreen01;
import worddatabase_using_architecture.ui.OutputActivity3;
import worddatabase_using_architecture.ui.OutputActivity4;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        // startActivity(new Intent(this, OutputOnMainThreadActivity.class));
        //  startActivity(new Intent(this, OutputActivity4.class));
        startActivity(new Intent(this, MovieScreen01.class));

    }
}

File: media_store_api/src/main/java/com/example/media_store_api/MainActivity.kt
package com.example.media_store_api

import android.app.Activity
import android.app.RecoverableSecurityException
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.core.app.ActivityCompat.startIntentSenderForResult
import readmedia.ReadMediaFiles

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val read = ReadMediaFiles(this)
//        read.readImage01Version1()
//        val remove=RemoveMediaFile(this)
//        val uri=Uri.parse("content://media/external/images/media/5801");
//        remove.removeImage01Version1(uri)


    }



    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        super.onActivityResult(requestCode, resultCode, data)
        if (resultCode == Activity.RESULT_OK && requestCode == 100) {

        }
    }


}
